examples/stable_diffusion_v2/schedulers.md

```diff
@@ -35,24 +35,27 @@ You can obtain diverse results according to the given **prompt**. Here are 5 exa
 | :----: | :----: | :----: | :----: | :----: |
 | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/6a84e5ae-baab-4461-91ec-14f55b16aee2" width="155" height="155" /> | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/62c0bb93-68a8-4ff1-8bc5-0c9305931ac0" width="155" height="155" /> | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/92441896-8786-465b-82a2-3438aad85476" width="155" height="155" /> | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/e626ae1b-8fa6-4ad8-a083-ec39e4944277" width="155" height="155" /> | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/d3728d81-4a31-4d53-baf0-cd79b6046ca3" width="155" height="155" /> |
 
-### Fast Inference
+### Changing the scheduler
 
-For fast inference, we provide several scripts using different schedulers:
+As is known, it is extremely difficult to measure quantitatively which scheduler works best for a given diffusion pipeline, so it is often recommended to simply try out which works best. You can test the stable diffusion model with different scheduler by the following
```

Comment:
"it is extremely difficult to measure quantitatively which scheduler works best for a given diffusion pipeline"  is improper.
As quantitative evaluation for different schedulers is usually not sufficient to determine which is the best, it is often recommended to simply try out and check out visually.